go/vt/vttablet/tabletserver/tabletserver.go

```diff
@@ -349,6 +349,7 @@ func (tsv *TabletServer) InitACL(tableACLConfigFile string, enforceTableACLConfi
 
 	if reloadACLConfigFileInterval != 0 {
 		ticker := time.NewTicker(reloadACLConfigFileInterval)
+		defer ticker.Stop()
```

Comment:
Don't we want this timer to keep running in the background to reload the acl config every `reloadACLConfigFileInterval` seconds?

Reply:
Hmm, you're right! Good catch! We have a never ending for loop below but that's in a goroutine so we don't want to stop the ticker when this outer function ends. Will change.